### PR TITLE
Blog post: CSL4LibreOffice

### PR DESCRIPTION
**Pull request by subhramit**
Refs https://github.com/JabRef/jabref/wiki/GSoC-2024-%E2%80%90-Improved-CSL-Support-(and-more-LibreOffice%E2%80%90JabRef-integration-enhancements)

### CODE REVIEW

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:32]:
```suggestion
1. Download the [development version of JabRef](https://builds.jabref.org/main/).
```

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:33]:
```suggestion
2. Connect to a running LibreOffice document instance from the [Libre/OpenOffice Panel](https://docs.jabref.org/cite/openofficeintegration)
```

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:59]: Add link to forum for feedback - otherwise, users might not know where to put feedback.

```suggestion
I hope this new feature enhances your research and writing process. As always, [we welcome your feedback and suggestions for further improvements](https://discourse.jabref.org/c/feedback/3).
```

Comment (koppor) [reply _posts/2024-08-23-GSoC-CSL.md:59]: Also move the paragraph as last.

Reason: Structure of a summary is:

1. Summary
2. Future work

**Review by koppor — CHANGES_REQUESTED** [review]

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:63]: Use passive voice in second part. "We" sounds strange, b/c all other parts are "I". Since you also welcome feedback *too* I suggest passive voice

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:17]: its --> JabRef's

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:23]: array? Maybe "collection"?

File: _posts/2024-08-23-GSoC-CSL.md
Comment (koppor) [inline _posts/2024-08-23-GSoC-CSL.md:24]: "Insert usual citiations"...

It reads strange that no attribute for "citations" first, but we have no word for "non-intext.citations", do we?

Replace "LaTeX's" by "natbib's" and link https://ctan.org/pkg/natbib.

**Review by koppor — CHANGES_REQUESTED** [review]
The more one reads, the more ideas one has 😅

File: _posts/2024-08-23-GSoC-CSL.md
Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: "Usual citations" sounds odd.
How about this ^ (check changes).

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: If not this, the other alternative - "insert citations directly into..." (remove in-text, as citations is generic and can encompass all types).

Comment (koppor) [reply _posts/2024-08-23-GSoC-CSL.md]: Then also remove link to natbib, because there is no distinction any more

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: Sorry, what I meant was, I have already changed it to:
"**Cite with Flexibility**: Insert citations (similar to natbib's `\cite`) as well as in-text citations (similar to natbib's and `\citet`) directly into your LibreOffice document."

If we don't like this, then we can go with:
"**Cite with Flexibility**: Insert citations directly into your LibreOffice document."
(with natbib removed).

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: I realized I forgot to add the link. Will wait for your approval (if we choose to go with the first one).

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: I think Thilo isn't tagged properly so @ThiloteE

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: @koppor Side comment: I do think case 2 looks cleaner (and more digestible for a general audience, even for students like me who didn't use LaTeX during the college course, the story of which I mentioned in the blog's introductory paragraph).

Comment (ThiloteE) [reply _posts/2024-08-23-GSoC-CSL.md]: I suggest trying to avoid filler words and non-words, such as "flexible", especially for documentation. I think the sentence is good enough for the Blog, though. While I have used LaTeX and bibaltex, I have never used the Natbib package before. I think JabRef and CSL define the way they cite themselves independently from Natbib, and, yes, while it can help users that are used to that package help understand the new features it might be easier to understand, if the feature is explained, with verbs instead of nouns.

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: Which sentence, Thilo?
Your suggestion would be to remove mentioning natbib (or LaTeX) altogether?
Are you referring to case 1, case 1 with natbib removed, or case 2?

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: I have updated it to:
"**Cite with ease**: Insert citations directly into your LibreOffice document."

Comment (ThiloteE) [reply _posts/2024-08-23-GSoC-CSL.md]: I am not sure what case 1 or case 2 are.

Ideally, I would prefer splitting up Cite and Cite-in-Text into separate sections, but it's fine, if that is explained more in detail in the documentation. 
The way it is now with natbib is ok. We have many LaTeX users, so Natbib might ring a bell.

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: Can you check if the current sentence is okay? Or if I should revert it and bring back the distinctions and natbib refrences?

Comment (ThiloteE) [reply _posts/2024-08-23-GSoC-CSL.md]: To be honest, I don't have a strong opinion here. Your choice :-)

Comment (subhramit) [reply _posts/2024-08-23-GSoC-CSL.md]: Alright.
@koppor Then I suggest the current going with the current:
"Cite with ease: Insert citations directly into your LibreOffice document."

**Comment by subhramit** [comment]
Hey, if everyone's alright with it, let's publish today (formally it's my last submission date)
@koppor @Siedlerchr @ThiloteE

**Comment by subhramit** [comment]
Hey, if everyone's alright with it, let's publish today (formally it's my last submission date)
@koppor @Siedlerchr @ThiloteE

**Review by Siedlerchr — APPROVED** [review]
from my point of view lgtm

File: _posts/2024-08-23-GSoC-CSL.md
Comment (ThiloteE) [inline _posts/2024-08-23-GSoC-CSL.md:33]: 2. Connect to a running LibreOffice document instance by either clicking the "Connect" or "Manual Connect" button in the [Libre/OpenOffice Panel](https://docs.jabref.org/cite/openofficeintegration) in the side pane.

File: _posts/2024-08-23-GSoC-CSL.md
Comment (ThiloteE) [inline _posts/2024-08-23-GSoC-CSL.md:32]: 1. Download the [development version of JabRef](https://builds.jabref.org/main/).

**Comment by subhramit** [comment]
@ThiloteE I have acted on your review. I have used "Click" instead of "Press" to keep consistency.

**Review by ThiloteE — APPROVED** [review]

**Review by koppor — APPROVED** [review]

**Pull request merged**
